### DISCUSSION TITLE
我设置了开机自启bvs refresh_token刷新问题

### DISCUSSION THREAD

**Discussion by gylang** [messense/aliyundrive-webdav#256] [Q&A] [answered]
我是用的是命令行方式，在刷新token的时候，refresh_token可能会重新刷新, 我的脚本无法获取到最新的refresh_token，导致启动服务失败，需要手动到网页上重新获取refresh_token。是不可以将其写入到文件进行持久化。

**Comment by messense** [comment, marked as answer]
https://github.com/messense/aliyundrive-webdav/issues/50#issuecomment-924834144

**Reply by gylang** [reply]
了解 谢谢
